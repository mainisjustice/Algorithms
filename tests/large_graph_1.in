10
0 14 3 19 16 14 2 0 4 9
14 0 18 13 4 15 3 14 18 16
3 18 0 13 14 14 2 13 7 19
19 13 13 0 8 14 11 7 7 6
16 4 14 8 0 10 8 0 4 19
14 15 14 14 10 0 0 7 8 15
2 3 2 11 8 0 0 10 0 0
0 14 13 7 0 7 10 0 14 3
4 18 7 7 4 8 0 14 0 0
9 16 19 6 19 15 0 3 0 0