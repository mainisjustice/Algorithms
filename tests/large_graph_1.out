0 5 3 11 8 12 2 12 4 9 
5 0 5 12 4 14 3 13 8 14 
3 5 0 13 9 14 2 12 7 12 
11 12 13 0 8 14 11 7 7 6 
8 4 9 8 0 10 7 15 4 14 
12 14 14 14 10 0 14 7 8 10 
2 3 2 11 7 14 0 10 6 11 
12 13 12 7 15 7 10 0 14 3 
4 8 7 7 4 8 6 14 0 13 
9 14 12 6 14 10 11 3 13 0 